### PR TITLE
Added board config for Mellow Fly E3 v2

### PR DESCRIPTION
**Pull request by LievenVanh**
Added new generic board configuration for Mellow Fly E3 v2

### CODE REVIEW

**Comment by JamesH1978** [comment]
Thankyou for submitting a PR, please can you sign off your as mentioned in point 3 in https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md#what-to-expect-in-a-review and provide am update to the printer.test file as per point 4 in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md#guidelines

Thanks
James

**Comment by LievenVanh** [comment]
> Thankyou for submitting a PR, please can you sign off your as mentioned in point 3 in https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md#what-to-expect-in-a-review and provide am update to the printer.test file as per point 4 in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md#guidelines
> 
> Thanks James

Sorry, I missed that completely, I'm pretty new to the collaboration stuff ;)
This should be fixed now

File: config/generic-mellow-fly-e3-v2.cfg [hunk 1]
```diff
@@ -0,0 +1,244 @@
+# Copyright (C) 2024 Lieven Vanhercke <lieven.vanh@gmail.com>
+
+# This file may be distributed under the terms of the GNU GPLv3 license.
```
Comment (JamesH1978) [inline config/generic-mellow-fly-e3-v2.cfg:3]: Please dont add your own copyright or details about licensing, this is covered by your commit, this is covered in viii in point 7 in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md#guidelines
James

File: config/generic-mellow-fly-e3-v2.cfg [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+########################################
+# TMC2209 configuration
+########################################
+
+[tmc2209 stepper_x]
+uart_pin: PC15
+interpolate: False
+run_current: 0.3
+sense_resistor: 0.110
+stealthchop_threshold: 500
```
Comment (JamesH1978) [inline config/generic-mellow-fly-e3-v2.cfg:122]: Please do not set a non 0 or 999999 figure for stealthchop_threshold, the option should be on or off. Please see https://github.com/Klipper3d/klipper/blob/master/docs/TMC_Drivers.md#setting-spreadcycle-vs-stealthchop-mode

This goes for all tmc sections

Thanks
James

File: config/generic-mellow-fly-e3-v2.cfg [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+control: pid
+pid_kp: 58.437
+pid_ki: 2.347
+pid_kd: 363.769
+
+#######################################
+# LIGHTING
+#######################################
+
+[led Toolhead]
```
Comment (JamesH1978) [inline config/generic-mellow-fly-e3-v2.cfg:173]: Please comment this section out, as it is not known if the end user will use this option.

Thanks
James

File: config/generic-mellow-fly-e3-v2.cfg [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+max_temp: 80
+
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 3000
+max_z_velocity: 50
+max_z_accel: 100
+
+[safe_z_home]
```
Comment (JamesH1978) [inline config/generic-mellow-fly-e3-v2.cfg:216]: This section is considered non-standard to a generic config, please remove it.

Thanks
James

File: config/generic-mellow-fly-e3-v2.cfg [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+home_xy_position:0,0
+speed:10
+z_hop:0
+
+[idle_timeout]
```
Comment (JamesH1978) [inline config/generic-mellow-fly-e3-v2.cfg:221]: Please do no disable or adjust safety options, please remove this section.

Thanks
James

**Review by JamesH1978 — CHANGES_REQUESTED** [review]
Please see my comments.

Thanks
James

**Comment by LievenVanh** [comment]
all changes are made, sorry for the extra work I am giving you :/

**Comment by KevinOConnor** [comment]
Hi.  I saw your comment on Discourse asking for a review, but it seems not all of James' comments have been addressed.

Cheers,
-Kevin

**Review by JamesH1978 — APPROVED** [review]
Looks fine to me.

Thanks
James

**Comment by KevinOConnor** [comment]
Oh - there is still the unusual copyright statement at the top of the config file.  Not a huge deal either way, but seems odd.

Cheers,
-Kevin

**Comment by JamesH1978** [comment]
Oh, I couldnt see the statement, i even went to his fork for the PR and its not there, unless i am missing it @KevinOConnor

File: config/generic-mellow-fly-e3-v2.cfg [hunk 6]
```diff
@@ -0,0 +1,234 @@
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
```
Comment (KevinOConnor) [inline config/generic-mellow-fly-e3-v2.cfg:2]: Typically example config start with a description, but the first two lines of this file have an unusual copyright statement.

-Kevin

Comment (JamesH1978) [reply config/generic-mellow-fly-e3-v2.cfg:2]: Oh you are totally right, in my head that was normal and in all configs for some reason! False memory

Comment (LievenVanh) [reply config/generic-mellow-fly-e3-v2.cfg:2]: Hi,

This is most likely the result of some copypasta. If you want I can remove the offending lines

**Review by JamesH1978 — APPROVED** [review]
Thanks for the change, looks good.

Thanks
James

**Pull request merged**